pkg/apis/planetscale/v2/vitesscluster_types.go

```diff
@@ -148,6 +152,17 @@ const (
 	ImmediateVitessClusterUpdateStrategyType VitessClusterUpdateStrategyType = "Immediate"
 )
 
+type ExternalVitessClusterUpdateStrategyOptions struct {
+	// AllowResourceChanges can be used to allow certain resource updates to the DataVolumeClaimTemplate
+	// to be updated immediately without using an external tool.
```

Comment:
This field is meant to be about resources in general, since we plan to support cpu/mem later, so it isn't specific to the data volume.

```suggestion
	// AllowResourceChanges can be used to allow changes to certain resource
	// requests and limits to propagate immediately, bypassing the external rollout tool.
```

Reply:
Done.